vehicle/mkvi/software/sensing/sensing.c

```diff
@@ -0,0 +1,73 @@
+/*
+Header:
+    Code for the Sensing Controller Board located in the nostril motherboard and
+the rear sensing box Author:
+    @author Richard Li
+    @date   03/25/2023
+*/
+
+/*----- Includes -----*/
+#include "sensing.h"
+#include "can_api.h"
+#include "log_uart.h"
+#include "utils/timer.h"
+#include <avr/interrupt.h>
+#include <avr/io.h>
+#include <stdio.h>
+#include <string.h>
+#include <util/delay.h>
+
+volatile uint8_t gTimerFlag = 0x01;
```

Comment:
What are `gTimerFlag` and `gStatusMessages`? Might be nice to include a comment about their function

Reply:
deleted from prev comment!